backend/routes/item-routes.js

```diff
@@ -7,47 +7,55 @@ const Item = require("../models/itemModel");
 const router = require("express").Router();
 
 require("dotenv").config();
-cloudinary.config({
-  cloud_name: "cloudtesting",
-  api_key: "465735684648442",
-  api_secret: "HVxIWBW7bQaBHJygz_qiprAfwok"
-});
+// cloudinary.config({
```

Comment:
Are these sections supposed to be commented out?

Reply:
i have it commented out since i am in the process of changing from multer to cloudinary-react, if i finalize that the frontend method is better, then ill completely delete them

Reply:
Got it, thanks for verifying :+1: